js/foundation.offcanvas.js

```diff
@@ -300,9 +380,15 @@ class OffCanvas extends Plugin {
     this.$triggers.attr('aria-expanded', 'false');
 
     if (this.options.trapFocus === true) {
-      this.$element.siblings('[data-off-canvas-content]').removeAttr('tabindex');
+      this.$content.removeAttr('tabindex');
       Keyboard.releaseFocus(this.$element);
     }
+
+    // Listen to transitionEnd and add class when done.
+    this.$element.one(Foundation.transitionend(this.$element), function(e) {
```

Comment:
This will need to change to using an import rather than Foundation global (see https://github.com/zurb/foundation-sites/blob/develop/js/foundation.util.motion.js#L4 for example)

but if you like I can do this at merge time.